Makefile

```diff
@@ -1,3 +1,4 @@
+# --- config and deploy ---
```

Comment:
`Configure and deploy` to keep consistent with `Install software` ?

Reply:
ack

Reply:
done